packages/markdown-it-bi-directional-links/src/utils.ts

```diff
@@ -32,7 +32,24 @@ export function genLink(
   md: MarkdownIt,
   href: string,
   link: RegExpMatchArray,
+  isInvalid = false,
 ) {
+  // Work when option noNoMatchedStillWork, like Wikipedia's invalid link
+  if (isInvalid) {
+    const openToken = state.push('link_open', 'a', 1)
+    openToken.attrSet('class', 'route-link')
```

Comment:
I suggest to add nolebase-route-link-invalid like class name for further styling.